app/constants/doc-collection-manifest.ts

```diff
@@ -0,0 +1,7 @@
+import { DocCollectionSource } from "./doc-collections.server"
+
+export const JSON_MANIFEST_FILENAME = "flow-docs.json"
```

Comment:
@10thfloor are you good with `docs/flow-docs.json` as the path for external repos to set these values?